src/wallet/rpcwallet.cpp

```diff
@@ -77,9 +77,9 @@ bool HaveKey(const SigningProvider& wallet, const CKey& key)
 
 bool GetWalletNameFromJSONRPCRequest(const JSONRPCRequest& request, std::string& wallet_name)
 {
-    if (request.URI.substr(0, WALLET_ENDPOINT_BASE.size()) == WALLET_ENDPOINT_BASE) {
+    if (URL_DECODE && request.URI.substr(0, WALLET_ENDPOINT_BASE.size()) == WALLET_ENDPOINT_BASE) {
```

Comment:
Shouldn't this be #ifdef URL_DECODE, on bitcoin-tx the symbol does not exist if libevent is not used.

Reply:
The symbol `URL_DECODE` is always defined

Reply:
my bad, thx